src/torch_ucc.cpp

```diff
@@ -469,16 +520,20 @@ void CommPG::enqueue_collective(
     std::unique_ptr<ProcessGroupUCC::WorkData> data,
     c10::intrusive_ptr<ProcessGroupUCC::WorkUCC> work,
     ucc_coll_args_t& coll,
-    ucc_team_h team) {
+    ucc_team_h team,
+    std::vector<ucp_ep_h> *eps,
+    int rank) {
   ucc_coll_req_h request;
   TORCH_UCC_CHECK(
       ucc_collective_init(&coll, &request, team), "failed to init collective");
   TORCH_UCC_CHECK(ucc_collective_post(request), "failed to post collective");
 
   auto entry =
-      std::make_shared<ProcessGroupUCC::ProgressEntry>(&ucc_comm, request);
+      std::make_shared<ProcessGroupUCC::ProgressEntry>(&ucc_comm, request, seq_num++);
```

Comment:
similar change should be done to `enqueue_cuda_collective` too (around line 512/567)